pom.xml

```diff
@@ -168,6 +187,11 @@
 			<artifactId>junit-vintage-engine</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter-engine</artifactId>
+			<scope>test</scope>
+		</dependency>
```

Comment:
I'd probably move this block up, so all Jupiter dependencies are grouped together.